workers/track-analysis/.env.example

```
@@ -0,0 +1,19 @@
+# Copy to workers/track-analysis/.env on worker machine.
+ANALYSIS_SERVICE_SECRET=replace-with-the-same-secret-used-in-vercel
+CONVEX_SITE_URL=https://your-deployment.convex.site
+SOUNDCLOUD_CLIENT_ID=
+SOUNDCLOUD_CLIENT_SECRET=
+SOUNDCLOUD_ACCESS_TOKEN=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SOUNDCLOUD_ACCESS_TOKEN key should go before the SOUNDCLOUD_CLIENT_ID key

(UnorderedKey)